compose/compose-file.md

```diff
@@ -409,6 +434,15 @@ options and tags it with the specified tag.
 > **Note**: In the [version 1 file format](compose-file.md#version-1), using `build` together
 > with `image` is not allowed. Attempting to do so results in an error.
 
+### isolation
+
+> [Added in version 2.1 file format](#version-21).
+
+Specify a container’s isolation technology. On Linux, the only supported value
+is `default`. On Windows, acceptable values are `default`, `process` and
+`hyperv`. Refer to the
+[Docker Engine docs](https://docs.docker.com/engine/reference/commandline/run/#specify-isolation-technology-for-container---isolation) for details.
```

Comment:
Can you change the URL to start at `/engine/` here? That way this link will work in staging, doc archives, etc. :)